src/web/nextui/src/app/eval/ResultsTable.tsx

```diff
@@ -647,28 +685,38 @@ function ResultsTable({
           ))}
         </thead>
         <tbody>
-          {reactTable.getRowModel().rows.map((row, rowIndex) => {
+          {reactTable.getRowModel().rows.map((row) => {
             let colBorderDrawn = false;
+
             return (
-              <tr key={row.id}>
+              <tr key={row.id} id={`row-${row.id}`}>
                 {row.getVisibleCells().map((cell) => {
                   const isMetadataCol =
                     cell.column.id.startsWith('Variable') || cell.column.id === 'description';
                   const shouldDrawColBorder = !isMetadataCol && !colBorderDrawn;
                   if (shouldDrawColBorder) {
                     colBorderDrawn = true;
                   }
-                  const shouldDrawRowBorder = rowIndex === 0 && !isMetadataCol;
```

Comment:
This removal seems to be unintentional?  It causes the table to change visually and draw a thicker border around every row